api/src/main/java/jakarta/faces/component/UIComponent.java

```diff
@@ -1842,4 +1841,41 @@ public void setTransient(boolean newTransientValue)
             }
         }
     }
+
+    private Resource getLocalizedCompositeResource(String resourceName, String libraryName, FacesContext context) 
+    {
+    	List<String> localizedPaths = getLocalizedPropertiesPaths(resourceName, context);
+    	Resource resource = null;
+    	
+    	for (String path_: localizedPaths) {
+    		resource = context.getApplication().getResourceHandler().createResource(path_, libraryName);
+    		if (resource != null)
```

Comment:
please dont use tabs and make { in new line
no idea, i thought our checkstyle would fail
also better name path_ into localizedPath

Reply:
Sorry about that. Fixed style in all four pull requests.